internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer_tasks.go

```diff
@@ -257,16 +257,6 @@ func DropCapabilities(seq runtime.Sequence, data interface{}) (runtime.TaskExecu
 			return fmt.Errorf("error setting secbits: %w", err)
 		}
 
-		// Set PR_SET_NO_NEW_PRIVS to limit setuid and similar privilege raising techniques.
-		// See https://www.kernel.org/doc/html/v5.10/userspace-api/no_new_privs.html.
```

Comment:
What do we lose by dropping this? Are we still protected from kexec?

Reply:
yeah, completely, this is not related to kexec. I tried to be smart, but I wasn't.

what this does it disables `allowPrivilegeEscalation` in Kubernetes terms. we would rather leave it to the user, as there are some valid usecases for it (`nginx-ingress` being one example).